test/users/test-edit.js

```diff
@@ -1,33 +0,0 @@
-const { assert } = require('chai');
```

Comment:
surely we want more tests for the new options, not fewer?

Reply:
This test didn't really test anything anyway. I'll add some new tests.